change/@fluentui-react-84e5924e-42c6-44ae-a13f-981d707ed986.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30221&commitId=5ea9732deaa4fba875912ad405ca390c5720dc4e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Split (compat)</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split (compat).Primary Disabled.hover split.chromium.png|1695| Changed|
|Button Split (compat).Primary Disabled.hover main.chromium.png|1695| Changed|
|Button Split (compat).Primary Disabled.default.chromium.png|1695| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>